mne/io/meas_info.py

```diff
@@ -321,23 +322,38 @@ def write_fiducials(fname, pts, coord_frame=0):
         the keys 'kind', 'ident' and 'r'.
     coord_frame : int
         The coordinate frame of the points (one of
-        mne.io.constants.FIFF.FIFFV_COORD_...)
+        mne.io.constants.FIFF.FIFFV_COORD_...).
     """
-    pts_frames = set((pt.get('coord_frame', coord_frame) for pt in pts))
-    bad_frames = pts_frames - set((coord_frame,))
-    if len(bad_frames) > 0:
-        err = ("Points have coord_frame entries that are incompatible with "
-               "coord_frame=%i: %s." % (coord_frame, str(tuple(bad_frames))))
-        raise ValueError(err)
+    write_dig(fname, pts, coord_frame)
 
-    fid = start_file(fname)
-    start_block(fid, FIFF.FIFFB_ISOTRAK)
-    write_int(fid, FIFF.FIFF_MNE_COORD_FRAME, coord_frame)
-    for pt in pts:
-        write_dig_point(fid, pt)
 
-    end_block(fid, FIFF.FIFFB_ISOTRAK)
-    end_file(fid)
+def write_dig(fname, pts, coord_frame=None):
```

Comment:
this is only for dig points, but not digitized electrode locs, right? that would just be written if you were writing out measurement info, correct?